core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -1043,6 +1043,11 @@ public List<Acl> getUniqueConsumerGroups(int tenantId) {
     return aclRepo.findAllByTenantId(tenantId);
   }
 
+  public boolean validateIfConsumerGroupUsedByAnotherTeam(
```

Comment:
@snuyanzin would be nice to have a unit test or @DataJpaTest for this query

Reply:
added